app_legacy/Helpers/render/game.php

```diff
@@ -195,7 +195,7 @@ function RenderGameAlts($gameAlts, $headerText = null): void
 {
     echo "<div class='component gamealts'>";
     if ($headerText) {
-        echo "<h3>$headerText</h3>";
+        echo "<h2 class='like-h3'>$headerText</h2>";
```

Comment:
To be in line with tailwind's naming I'd go with `text-h3`.

Reply:
You are right, `text-h3` would sound better and fit nice with rest of Tailwind's class names (I'm just used to name it `like-h3` as this is what I do often at work, where is no Tawilwind). Changed the names everywhere I used them.